crypto/verifier.go

```diff
@@ -49,8 +61,19 @@ func PublicKeyFromPEM(pemEncodedKey string) (crypto.PublicKey, error) {
 	return parsedKey, nil
 }
 
+// VerifyObject verifies the output of SignObject.
```

Comment:
nit: "verifies the output of SignObject". I'm not sure what that means. Is SignObject == obj? Please rephrase. We should probably mention that obj is marshaled to JSON and hashed with CommonJSONHash before verification.